wcfsetup/install/files/lib/system/style/command/CreateManifest.class.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace wcf\system\style\command;
+
+use wcf\data\page\PageCache;
+use wcf\data\style\Style;
+use wcf\system\io\AtomicWriter;
+use wcf\system\language\LanguageFactory;
+use wcf\system\WCF;
+use wcf\util\JSON;
+
+/**
+ * Generate then `manifest-*.json` files for a style.
+ *
+ * @author      Olaf Braun
+ * @copyright   2001-2024 WoltLab GmbH
+ * @license     GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since       6.1
+ */
+final class CreateManifest
+{
+    private readonly int $styleID;
+
+    public function __construct(Style $style)
+    {
+        $this->styleID = $style->styleID;
```

Comment:
There is no need to fetch the style again, just store the passed in Style object.